### ISSUE TITLE
Issues on Windows

### ISSUE THREAD

**Issue by mattcockrell** [IvanJosipovic/KubeUI#825]
I'm having an issue running the application on Windows 11.  The background process will start/restart about 3 times, then eventually close.  No UI is ever displayed.

**Comment by IvanJosipovic** [comment]
Hey, does kubectl work for you in terminal?

I see some errors in the logs about not being able to load the kubeconfig. Though I have no way to know they came from you :)

If so, would you be able to post a redacted version of the kubeconfig (redact password etc)

Alternatively, you can enable local file logging and post the errors here:
- Open C:\Users\\{username}\\.kubeui\settings.json
- Set  "LoggingEnabled": true,
- A fIle log will be created in C:\Users\\{username}\\.kubeui\app.log

**Comment by mattcockrell** [comment]
That may be me.  I don't set a value for the global kubeconfig setting.  I just bring up a client-specific config within my terminal sessions.  I'll try setting a value to see if the app will launch.

Would it be possible to support a blank kubeconfig?

**Comment by IvanJosipovic** [comment]
> Would it be possible to support a blank kubeconfig?

For sure, I will add some more test around this shortly.

**Comment by mattcockrell** [comment]
```
2024-12-12T17:32:03.1933274-05:00	INFO	[Microsoft.Hosting.Lifetime]	[0]	Application started. Press Ctrl+C to shut down.
2024-12-12T17:32:03.2005091-05:00	INFO	[Microsoft.Hosting.Lifetime]	[0]	Hosting environment: Production
2024-12-12T17:32:03.2005764-05:00	INFO	[Microsoft.Hosting.Lifetime]	[0]	Content root path: C:\Users\xxx\AppData\Local\KubeUI\current
2024-12-12T17:32:03.6983157-05:00	INFO	[KubeUI.App]	[0]	Application Started
2024-12-12T17:32:04.2906003-05:00	CRIT	[KubeUI.App]	[0]	Unhandled ExceptionSystem.NullReferenceException: Object reference not set to an instance of an object.
   at KubeUI.Client.ClusterManager.LoadFromConfigFromPath(String path)
   at KubeUI.Client.ClusterManager.LoadClusters()
   at KubeUI.Client.ClusterManager..ctor(ILogger`1 logger, IServiceProvider serviceProvider, IFactory factory)
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodBaseInvoker.InvokeDirectByRefWithFewArgs(Object obj, Span`1 copyOfArgs, BindingFlags invokeAttr)
   at System.Reflection.MethodBaseInvoker.InvokeWithFewArgs(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.RuntimeConstructorInfo.Invoke(BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSiteMain(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitRootCache(ServiceCallSite callSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.Resolve(ServiceCallSite callSite, ServiceProviderEngineScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.CreateServiceAccessor(ServiceIdentifier serviceIdentifier)
   at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService[T](IServiceProvider provider)
   at KubeUI.Utilities.GetRequiredService[T](Application app)
   at KubeUI.ViewModels.MainViewModel..ctor()
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)
   at System.Reflection.RuntimeConstructorInfo.Invoke(BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSiteMain(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitDisposeCache(ServiceCallSite transientCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.Resolve(ServiceCallSite callSite, ServiceProviderEngineScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.DynamicServiceProviderEngine.<>c__DisplayClass2_0.<RealizeService>b__0(ServiceProviderEngineScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService[T](IServiceProvider provider)
   at KubeUI.App.OnFrameworkInitializationCompleted()
   at Avalonia.AppBuilder.SetupUnsafe()
   at Avalonia.AppBuilder.Setup()
   at Avalonia.AppBuilder.SetupWithLifetime(IApplicationLifetime lifetime)
   at Avalonia.ClassicDesktopStyleApplicationLifetimeExtensions.StartWithClassicDesktopLifetime(AppBuilder builder, String[] args, Action`1 lifetimeBuilder)
   at KubeUI.Desktop.Program.Main(String[] args)

2024-12-12T17:32:05.4383290-05:00	INFO	[Microsoft.Hosting.Lifetime]	[0]	Application is shutting down...

```

**Comment by mattcockrell** [comment]
We also use kubelogin to auth through the Azure portal, in case KubeUI is also expecting to have a token in the kubeconfig.

**Comment by IvanJosipovic** [comment]
This issue should be resolved in 1.0.0-alpha.90


> We also use kubelogin to auth through the Azure portal, in case KubeUI is also expecting to have a token in the kubeconfig.

Kubelogin is supported, I use this tool with AKS daily.
